services/ui/src/components/Project/index.js

```diff
@@ -68,6 +72,12 @@ class Project extends React.Component {
               <div className='field'>{this.props.project.pullrequests}</div>
             </div>
           </div>
+          <div className='field-wrapper envlimit'>
+            <div>
+              <label>Non-Production environments in use</label>
```

Comment:
```suggestion
              <label>Development environments in use</label>
```

Let's call them what they are: `Development Environments`, I would like to not introduce a new word "non-production" when they are called "development" in the API.